Meta/CMake/vcpkg/overlay-ports/angle/fix-freebsd.patch

```diff
@@ -0,0 +1,56 @@
+diff --git a/src/common/platform.h b/src/common/platform.h
+index 44f485f1ce..fbbbfa7919 100644
+--- a/src/common/platform.h
++++ b/src/common/platform.h
+@@ -23,7 +23,10 @@
+ #elif defined(__linux__) || defined(EMSCRIPTEN)
+ #    define ANGLE_PLATFORM_LINUX 1
+ #    define ANGLE_PLATFORM_POSIX 1
+-#elif defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__) ||              \
++#elif defined(__FreeBSD__)
```

Comment:
does this patch match how angle is patched in chromium/other angle user ports in the FreeBSD ports tree? It looks pretty reasonable either way.

Reply:
This is very close to the patches from the angle of the FreeBSD chromium port. AFAICT getting angle to work properly will require more patches but this should be fine for now and there really isn't a way to test it either way.